testdata/src/retry.plain.summary.log

```diff
@@ -0,0 +1,35 @@
+
+Failed scenarios:
+samples/retry/retry.feature:17 # Test cases won't retry after failing more than the --retry limit
+
+Ambiguous scenarios:
+samples/retry/retry.feature:20 # Test cases won't retry when the status is AMBIGUOUS
+
+Pending scenarios:
+samples/retry/retry.feature:23 # Test cases won't retry when the status is PENDING
+
+Undefined scenarios:
+samples/retry/retry.feature:26 # Test cases won't retry when the status is UNDEFINED
+
+7 Scenarios (3 passed, 1 pending, 1 undefined, 1 ambiguous, 1 failed)
+12 Steps (3 passed, 1 pending, 1 undefined, 1 ambiguous, 6 failed)
```

Comment:
@davidjgoss how should retried steps be counted? Does the retry replace the original execution?